monai/losses/image_dissimilarity.py

```diff
@@ -81,10 +83,15 @@ def __init__(
                 - ``"sum"``: the output will be summed.
             smooth_nr: a small constant added to the numerator to avoid nan.
             smooth_dr: a small constant added to the denominator to avoid nan.
+
+        .. deprecated:: 0.6.0
+            ``ndim`` is deprecated, use ``spatial_dims``.
         """
         super(LocalNormalizedCrossCorrelationLoss, self).__init__(reduction=LossReduction(reduction).value)
 
-        self.ndim = ndim
+        if ndim is not None:
+            spatial_dims = ndim
```

Comment:
What if user set both `ndim` and `spatial_dims`?

Thanks.

Reply:
I would say that the new shall have higher priority, so the condition shall be if `ndim` is not default and `spatial_dims` is default

Reply:
In the current implementation we couldn't detect if it's a default spatial dim vs a user specified one though (when the value is 3)

Reply:
OK, I see your point.

Thanks.